File: golang/read_json_file.go
package main

import (
	"encoding/json"
	"fmt"
	"os"
)

func main() {
	filePath := "people.json"

	file, _ := os.Open(filePath)
	defer file.Close()

	var people []map[string]interface{}
	decoder := json.NewDecoder(file)
	decoder.Decode(&people)

	for _, person := range people {
		age := person["age"].(float64)
		firstName := person["first_name"].(string)
		fmt.Printf("Hello, %.0f year old %s\n", age, firstName)
	}
}

File: rust/read_json_file.rs
//cargo-deps: json="0.12.4"
use std::fs;

extern crate json;

fn main() {
    let filename = "people.json";

    let json_string = fs::read_to_string(filename).unwrap();
    let parsed_json = json::parse(&json_string).unwrap();

    for person in parsed_json.members() {
        let age = person["age"].as_u32().unwrap();
        let first_name = person["first_name"].as_str().unwrap();
        println!("Hello, {} year old {}", age, first_name);
    }
}
